exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,14 @@
+export default (n) => {
+  if (n <= 0) {
+    throw new Error('Only positive numbers are allowed');
+  }
+
+  const iter = (num, acc) => {
```

Comment:
It would be ok to write longer variable names: `number` instead of `num`; `iteration` or `iterate` instead of `iter`.

What would be a better name for `acc`? `count`? `steps`? Any idea?